front/admin/cli.ts

```diff
@@ -571,6 +573,44 @@ const productionCheck = async (command: string, args: parseArgs.ParsedArgs) => {
   }
 };
 
+async function apikeys(command: string, args: parseArgs.ParsedArgs) {
+  switch (command) {
+    case "set-role": {
+      if (!args.wId) {
```

Comment:
This is expeted to be a string. Look for `Authenticator.internalAdminForWorkspace` to get an authenticated from it